content/tutorials/git_authentication/index.Rmd

```diff
@@ -80,12 +91,19 @@ Anyone who has your token, has access to your GitHub account.
 
 Check if everything is OK:
 
-```{r eval=FALSE}
+```{r}
 usethis::git_sitrep()
 ```
 
 The default Git protocol should be 'https' (recommended on Windows).
 
+The `usethis` package promotes the use of a global `.gitignore` file which prevents that some file types that could contain sensitive information (from your account / credentials) are tracked by the Git version control system.
+However, we prefer a [project-specific `.gitignore` file](https://github.com/inbo/checklist/blob/main/inst/generic_template/gitignore) for this purpose.
+This is one of the many things that the `checklist` package will take care off for you.
```

Comment:
typo: off -> of

Reply:
[5c2e40b](https://github.com/inbo/tutorials/pull/309/commits/5c2e40bac7d4be372c08f9d9971cd9ee594adc62)